jobs/JGLOBAL_ARCHIVE_TARS

```diff
@@ -1,6 +1,6 @@
 #! /usr/bin/env bash
 
-source "${HOMEgfs}/ush/jjob_header.sh" -e "arch_tars" -c "base arch_tars wave"
+source "${HOMEgfs}/ush/jjob_header.sh" -e "arch_tars" -c "base arch_tars"
```

Comment:
Was "wave" here erroneously?

Reply:
Pinging @DavidHuber-NOAA  - I forget the exact reason but it was put there intentionally.   

It does seem silly for gcafs to need a config.wave though...

Reply:
My suggestion is the following:
```
source "${HOMEgfs}/ush/jjob_header.sh" -e "arch_vrfy" -c "base arch_vrfy"
if [[ "${DO_WAVE}" == "YES" ]]; then
  source "${EXPDIR}/config.wave" 
  source "${USHgfs}/wave_domain_grid.sh"
fi
```

Reply:
Ok, works for me. I can incorporate this into a commit when I address Kate's comments about the OBSDMP variable

Reply:
@DavidHuber-NOAA I just thought. Where is $DO_WAVE defined? Isn't it in `base`? Then how can this if be done this way?